packages/@react-spectrum/s2/stories/Tabs.stories.tsx

```diff
@@ -29,65 +30,99 @@ const meta: Meta<typeof Tabs> = {
+        </div>
+      </TabPanel>
+      <TabPanel id="Emp" UNSAFE_style={{display: 'flex'}} data-panel>
+        <div style={{overflow: 'auto'}}>
+          <p>Alea jacta est.</p>
+        </div>
+      </TabPanel>
+    </Tabs>
+  </div>
 );
 
 export const Disabled = (args: any) => (
-  <Tabs {...args} styles={style({width: '[450px]', height: 144})} disabledKeys={['FoR', 'MaR', 'Emp']}>
-    <TabList aria-label="History of Ancient Rome">
-      <Tab id="FoR"><Edit />Founding of Rome</Tab>
-      <Tab id="MaR">Monarchy and Republic</Tab>
-      <Tab id="Emp">Empire</Tab>
-    </TabList>
-    <TabPanel id="FoR">
-      Arma virumque cano, Troiae qui primus ab oris.
-    </TabPanel>
-    <TabPanel id="MaR">
-      Senatus Populusque Romanus.
-    </TabPanel>
-    <TabPanel id="Emp">
-      Alea jacta est.
-    </TabPanel>
-  </Tabs>
+  <div className={style({width: '[700px]', height: 256, resize: 'horizontal', overflow: 'hidden', padding: 8})}>
+    <Tabs {...args} styles={style({width: 'full'})} disabledKeys={['FoR', 'MaR', 'Emp']}>
+      <TabList aria-label="History of Ancient Rome">
+        <Tab id="FoR"><Edit /><Text>Founding of Rome</Text></Tab>
+        <Tab id="MaR">Monarchy and Republic</Tab>
+        <Tab id="Emp">Empire</Tab>
+      </TabList>
+      <TabPanel id="FoR">
+        Arma virumque cano, Troiae qui primus ab oris.
+      </TabPanel>
+      <TabPanel id="MaR">
+        Senatus Populusque Romanus.
+      </TabPanel>
+      <TabPanel id="Emp">
+        Alea jacta est.
+      </TabPanel>
+    </Tabs>
+  </div>
 );
 
 export const Icons = (args: any) => (
-  <Tabs {...args} styles={style({width: 208, height: 144})}>
-    <TabList aria-label="History of Ancient Rome">
-      <Tab id="FoR" aria-label="Edit"><Edit /></Tab>
-      <Tab id="MaR" aria-label="Notifications"><Bell /></Tab>
-      <Tab id="Emp" aria-label="Likes"><Heart /></Tab>
-    </TabList>
-    <TabPanel id="FoR">
-      Arma virumque cano, Troiae qui primus ab oris.
-    </TabPanel>
-    <TabPanel id="MaR">
-      Senatus Populusque Romanus.
-    </TabPanel>
-    <TabPanel id="Emp">
-      Alea jacta est.
-    </TabPanel>
-  </Tabs>
+  <div className={style({width: '[700px]', height: 256, resize: 'horizontal', overflow: 'hidden', padding: 8})}>
```

Comment:
I noticed that for this story the tab collapses to its full title version instead of retaining the icon only: https://reactspectrum.blob.core.windows.net/reactspectrum/e76be75c457734e4f5b5f322dfe7b0a10b15ecd5/storybook-s2/index.html?path=/story/tabs--icons

Reply:
You mean that it has titles in the picker and in the drop down menu?

This is an open question, how do we want this to work? Should the picker selected value just be the icon? the menu would maybe look out of place (very wide and with the text)? we'll also possibly need to do some work to get the labelling correct on the picker. I haven't looked into that yet.

Reply:
Oh I see. I think at the very least that the selected value in the picker should still have the icon alongside the text, but as for the text being there in the first place, I think I actually prefer it now.

Reply:
I'm not sure what you mean? it is beside the text?
<img width="210" alt="Screenshot 2024-11-01 at 4 54 55 pm" src="https://github.com/user-attachments/assets/1a56f957-2461-4d2b-87d9-edbb65aa01f5">

Reply:
Hm this is what I see [here](https://reactspectrum.blob.core.windows.net/reactspectrum/e76be75c457734e4f5b5f322dfe7b0a10b15ecd5/storybook-s2/index.html?path=/story/tabs--icons) when collapsed: 
<img width="481" alt="image" src="https://github.com/user-attachments/assets/d7803f89-c307-4dd3-959c-45d3f40553d9">

Reply:
Huh... clear the cache? Mine is chrome as well

Reply:
yep, cache issue, resolving